e2e/features/idam-sign-in.feature

```diff
@@ -1,10 +1,5 @@
 Feature: User authentication
 
-  Scenario: Should show Sign In form instead of Case List when user has *not* signed in
```

Comment:
What is this feature and is it relevant for Bulk Scanning?

Reply:
removed 👍